src/main/java/org/codehaus/mojo/taglist/TagListReport.java

```diff
@@ -525,11 +533,7 @@ String getExcludesCommaSeparated() {
      *
      * @return The Locale of the source files.
      */
-    public Locale getLocale() {
-        // The locale string should never be null.
-        if (sourceFileLocale == null) {
-            sourceFileLocale = DEFAULT_LOCALE;
-        }
+    public Locale getSourceFileLocale() {
         return new Locale(sourceFileLocale);
```

Comment:
Note that this one is a bug. The one-arg ctor expects a *language*, not a locale. `de_AT` will fail or result in undefined behavior.  You need something like this: https://github.com/apache/maven-doxia-sitetools/blob/ea6eb76daffa47b4b4fa276fc89b07213c9d7107/doxia-integration-tools/src/main/java/org/apache/maven/doxia/tools/DefaultSiteTool.java#L736

Reply:
done